modules/installation/proc-config-rhdh-custom-resource.adoc

```diff
@@ -1,4 +1,53 @@
 [id="proc-config-rhdh-custom-resource_{context}"]
 = Configuring the {product-short} Custom Resource
 
-Updates to the Backstage custom resource (CR) are automatically handled by the {product} Operator. However, updates to resources referenced by the CR, such as ConfigMaps or Secrets, are not updated automatically unless the CR itself is updated. If you want to update resources referenced by the CR, then you must manually delete the Backstage Deployment so that the Operator can re-create it with the updated resources.
\ No newline at end of file
+You can change the configuration of the Custom Resource (CR) to program the way that the Operator manages your {product} instance.
+
+[NOTE]
```

Comment:
I think we can now remove this note. For 1.2 (now that https://issues.redhat.com/browse/RHIDP-1425 (https://github.com/janus-idp/operator/issues/236) is done), this is no longer true. By default, the Operator will automatically restart the RHDH instance upon changes to referenced ConfigMaps/Secrets.
cc @gazarenkov

Reply:
@rm3l this is an intro to explain what this procedure is and why a user might want or need to do this procedure. Can you help me fill in the user story? 

As a [role], I want to change the configuration of the CR so that I can [???].

Reply:
Ah, good to know, thank you! 

Note: I formatted this text as a Note, but it's currently the entirely of the _Configuring the RHDH CR_ section published in the Admin guide (https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.1/html/administration_guide_for_red_hat_developer_hub/assembly-install-rhdh-ocp#proc-config-rhdh-custom-resource_admin-rhdh). So it's good that we're repurposing that section with new (relocated) content.